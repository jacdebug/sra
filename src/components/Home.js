import React from 'react';
import { NavLink } from 'react-router-dom';

const Home = () => <NavLink to="/files?_page=1">My Files</NavLink>;

export default Home;
