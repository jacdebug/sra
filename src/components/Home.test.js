import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <BrowserRouter>
      <Home />
    </BrowserRouter>,
    div
  );
});

it('renders a snapshot', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
