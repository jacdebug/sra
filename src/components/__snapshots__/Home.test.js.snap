// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<a
  className={undefined}
  href="/files?_page=1"
  onClick={[Function]}
  style={undefined}
>
  My Files
</a>
`;
